src/content/components/Router/Router.js

```diff
@@ -355,6 +356,15 @@ export class Router extends React.PureComponent {
           component: MyBugs,
         },
       },
+      {
+        label: "General Triage",
```

Comment:
I don't know how I feel about `General Triage` - I think with the context of the subheading it makes sense but without it not so much? But I guess everyone who's using this Bugzy view would have that context so maybe this is irrelevant (and I have no suggestions lol).

Reply:
Yeah also I think most likely every mozilla engineer would know about general triage since it's mentioned in the biweekly desktop meeting notes. And to be fair, some of the other tab names seem pretty obscure until you click them haha